### ISSUE TITLE
Tarballs are untested in CI

### ISSUE THREAD

**Issue by SpamapS** [gearman/gearmand#357]
As important as building from git is, the gold standard for portability and release artifacts is still tarballs, I think.  Maybe I'm wrong? Anyway, if we are going to keep releasing via tarball and not just git tag, we should build a tarball in CI. 

It would be helpful if users could let us know if they use the tarballs in any way.

**Comment by esabol** [comment]
Yeah, makes sense. I'm pretty sure most people build from the release tarball if they are compiling gearmand at all. These days, most folks `apt-get install` (or equivalent) whatever package is provided for their system, I presume, but those packages track our release tarballs, not the git repository.

Adapting the GitHub Actions CI workflow to generate a tarball and then build and test that tarball sounds tricky, but I might be overthinking it.
